File: push/src/main/java/com/xiaomi/smack/PacketListener.java
package com.xiaomi.smack;

import com.xiaomi.slim.Blob;
import com.xiaomi.smack.packet.Packet;

/* JADX INFO: loaded from: miuipushsdkshared_3_7_9.jar:com/xiaomi/smack/PacketListener.class */
public interface PacketListener {
    void process(Blob blob);

    void processPacket(Packet packet);
}

File: legacy/src/main/java/com/xiaomi/smack/PacketListener.kt
package com.xiaomi.smack
import io.github.magisk317.mipush.protocol.model.*

import com.xiaomi.slim.Blob
import com.xiaomi.smack.packet.Packet

interface PacketListener {
    fun process(blob: Blob)

    fun processPacket(packet: Packet)
}
